crates/router/src/core/admin.rs

```diff
@@ -830,6 +831,218 @@ pub fn get_frm_config_as_secret(
     }
 }
 
+pub async fn create_business_profile(
+    db: &dyn StorageInterface,
+    request: api::BusinessProfileCreate,
+    merchant_id: &str,
+) -> RouterResponse<api_models::admin::BusinessProfileResponse> {
+    let key_store = db
```

Comment:
Make the db calls parallel.

Reply:
find_merchant_account depends on keystore, cannot make these parallel